nidirect_media/nidirect_media.module

```diff
@@ -136,3 +138,22 @@ function nidirect_media_form_alter(&$form, FormStateInterface $form_state, $form
     }
   }
 }
+
+/**
+ * Implements hook_preprocess_views_view_field().
+ *
+ * Injects the usage value of a media item into the table display.
+ */
+function nidirect_media_preprocess_views_view_field(&$variables) {
+  $view = $variables['view'];
+  $field_id = $variables['field']->field;
+
+  if ($view->id() == 'media' && $view->current_display == 'media_page_list' && $field_id == 'nothing') {
+    $mid = $variables['row']->mid;
+    $usage = \Drupal::service('whatlinkshere.linkmanager')->getReferenceContent($variables['row']->_entity, PHP_INT_MAX, 0);
```

Comment:
`PHP_INT_MAX` used because this service function is sort of shoehorned/repurposed here, and I just needed a suitably large number.